### ISSUE TITLE
Error in plot_summs function in v1.0.0

### ISSUE THREAD

**Issue by tkhartman** [jacob-long/jtools#28]
The latest update to jtools (v.1.0.0) now creates error when trying to use the plot_summs function. For instance, this simple example demonstrates the issue:

`require('jtools')
set.seed(1234)
y <- rbinom(n = 1000, size = 1, prob = 0.4)  
x <- rbinom(n = 1000, size = 1, prob = 0.5)  
df <- data.frame(x, y)  
model1 <- glm(y ~ x, family = binomial(link = "logit"), data = df)  
plot_summs(model1, exp = TRUE)`

> Error in stop_wrap("Install the ggstance package to use the plot_coefs function.") : 
  Install the ggstance package to use the plot_coefs function.

Installing ggstance solves the problem on my OSX machine but not a colleague's PC running windows, which then errors with the following:

> Error in getS3method("tidy", method_stub, envir = getNamespace("broom")) : 
  unused argument (envir = getNamespace("broom"))

(Un)reinstalling broom does not solve the issue.

**Comment by jacob-long** [comment]
Thanks for reporting this. It doesn't replicate on my machine, though I am using macOS as well. I will try this on my Windows 10 desktop soon to see if this is a platform-specific thing or if it is just a coincidence that there is a difference on your colleague's Windows PC.

**Comment by sunyixian** [comment]
I have the same problem and hope we can find a solution soon...

**Comment by jacob-long** [comment]
Based on the error message, I'm inclined to think it may be similar to this: https://github.com/tidyverse/tidyr/issues/202 

That would mean @tkhartman's colleague is using a rather old version of R (R 3.3.0 or older it appears). I can implement a fix that would make it work, but I'll only bother if that's actually the problem.

@sunyixian, are you getting this same error message? 

```
Error in getS3method("tidy", method_stub, envir = getNamespace("broom")) : 
  unused argument (envir = getNamespace("broom"))
```

If so, could you tell me what version of R you are using, what OS your computer is (Windows/macOS/Linux)?

**Comment by sunyixian** [comment]
@jacob-long Yes, I got that same error message. I just checked my R version and realized that I was using a very old version (3.2.4). After all necessary updates, the new package works. Thank you!
